R/apply_frmt_methods.R

```diff
@@ -154,6 +154,9 @@ apply_frmt.frmt_combine <- function(frmt_def, .data, values, mock = FALSE, param
     str_extract_all("(?<=\\{)[^\\}]+(?=\\})") %>%
     unlist()
 
+  # Adding the unquoted version to match while long
+  fmt_param_vals_uq <- str_remove_all(fmt_param_vals, "`")
+
   # Check if unspecified param values are in the dataset
 
   if(!setequal(names(frmt_def$fmt_ls), fmt_param_vals)){
```

Comment:
Should fmt_param_vals be changed to fmt_param_vals_uq?

Reply:
We end up needing both. When you are filtering in the data before it goes wide it needs to be without the `, but when you are doing things with the new columns you need `. So there isn't any winning